src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,17 @@
+package duke.task;
+
+import duke.task.Task;
+
+public class Event extends Task {
+    protected String at;
```

Comment:
This naming can suggest a location or time property. You could try another name that could increasing meaning here. See [https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#not-too-long-not-too-short](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#not-too-long-not-too-short).